geo_json_metrics/vcr_calculator.py

```diff
@@ -0,0 +1,23 @@
+def map_vcr(velocities: list[float]) -> list[float]:
+    """Map the VCR function over each pair from the list, skipping the first element
+    Args:
+        velocities (list[float]): A list of floats corresponding to the velocity (technically the speed since there is no
+        vector describing the direction)
+    Returns:
+        list[float]: new list of Velocity Change Rates (the list is length of inputlist - 1)
+    """
```

Comment:
Hvad skal jeg forvente som output, som input listen består af 0 eller 1 element?
En test til disse to cases ville være godt :)

Reply:
Kaster nu en TooFewElementsException